### PR TITLE
Added the possibility to use comments in the dump file

### PR DESCRIPTION
**Pull request by oriceon**
added useSkipComments() and dontUseSkipComments()

### CODE REVIEW

File: src/Databases/MySql.php [hunk 1]
```diff
@@ -60,6 +63,26 @@ public function dontUseSingleTransaction()
...
+    {
+        $this->useSkipComments = true;
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function dontUseSkipComments()
```
Comment (freekmurze) [inline src/Databases/MySql.php]: This is the default right? 

If so, remove this method.

File: src/Databases/MySql.php [hunk 2]
```diff
@@ -99,12 +122,15 @@ public function dumpToFile(string $dumpFile)
     public function getDumpCommand(string $dumpFile, string $temporaryCredentialsFile): string
     {
         $command = [
-            "'{$this->dumpBinaryPath}mysqldump'",
+            "\"{$this->dumpBinaryPath}mysqldump\"",
```
Comment (freekmurze) [inline src/Databases/MySql.php]: This change breaks all tests, please remove the change.

File: src/Databases/MySql.php [hunk 3]
```diff
@@ -60,6 +63,26 @@ public function dontUseSingleTransaction()
     }
 
     /**
+     * @return $this
+     */
+    public function useSkipComments()
```
Comment (freekmurze) [inline src/Databases/MySql.php]: I think `skipComments` sound friendlier than `useSkipComments`. If you agree, please rename all usages.

**Comment by freekmurze** [comment]
Thanks for your contribution. I've added some remarks in the code.

**Comment by oriceon** [comment]
Then why you use dontUseExtendedInserts ? We should remove it too ?

**Comment by freekmurze** [comment]
In that case, ignore my comment, and leave `dontSkipComments` in there, to be consistent.

**Comment by oriceon** [comment]
And about ```"'{$this->dumpBinaryPath}mysqldump'",``` this will breake command to run in Windows. This is the reason that i edited into "\"command\""

**Comment by freekmurze** [comment]
With the quotes present, it does not work on Linux systems...

**Comment by oriceon** [comment]
@freekmurze ok i added a quote selection depends on platform..

**Pull request closed without merging**